source/javascripts/refills/coffeescript/expander.coffee

```diff
@@ -1,9 +1,7 @@
 $(document).ready ->
-  expanderTrigger = document.getElementById("js-expander-trigger")
-  expanderContent = document.getElementById("js-expander-content")
-  $("#js-expander-trigger").click ->
-    $(this).toggleClass "expander-hidden"
+  expanderTrigger = $('.expander-trigger')
+  expanderContent = $('.expander-content')
```

Comment:
Right, I kept it because it was already there. 

In general, it looks like the Refills JavaScript snippets could stand some code review for things like this. If I have time, I'll submit some revisions.